packages/flutter/test/material/dropdown_test.dart

```diff
@@ -452,6 +507,27 @@ void main() {
         buttonBox.size.centerRight(Offset(-arrowIcon.size.width, 0.0)).dx);
   });
 
+  testWidgets('Arrow icon aligns with the edge of button in form field when expanded', (WidgetTester tester) async {
+    final Key buttonKey = UniqueKey();
+
+    // There shouldn't be overflow when expanded although list contains longer items.
+    final List<String> items = <String>['1234567890','abcdefghijklmnopqrstuvwxyz1234567890abcdefghijklmnopqrstuvwxyz1234567890'];
+
+    Widget build() => buildFormFrame(buttonKey: buttonKey, value: '1234567890', isExpanded: true, onChanged: onChanged, items: items);
```

Comment:
This can be just inserted directly into `pumpWidget`, I don't think this needs to be a separate line.